configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -1,106 +1,32 @@
-#
-########################## types ########################
-#
-types:
-# QUOTA
-- name: ratelimit_by_service_by_consumer
-  quota_params:
-    dimensions:
-      service: STRING
-      consumer: STRING
-    maxAmount: 500
-    expiration: 1s
-- name: ratelimit_by_service_by_datacenter
-  quota_params:
-    dimensions:
-      service: STRING
-      datacenter: STRING
-    maxAmount: 10000
-    expiration: 1s
-
-# LIST
-- name: versionListChecker
-  mylistchecker_params: # EMPTY
-
-# METRIC
-- name: istio_time_distribution
-  metric_params:
-    description: "Distribution metric with Istio standard dimensions and buckets appropriately-sized for monitoring latency"
-    kind: DISTRIBUTION
-    value: DURATION
-    buckets:
-      explicit_buckets:
-        bounds: [0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10]
-    dimensions:
-      source: STRING
-      target: STRING
-      service: STRING
-      method: STRING
-      version: STRING
-      response_code: INT64
-- name: istio_size_distribution
-  metric_params:
-    description: "Distribution metric with Istio standard dimensions and buckets appropriately-sized for monitoring sizes of requests/responses"
-    kind: DISTRIBUTION
-    value: INT64
-    buckets:
-      exponentialBuckets:
-        numFiniteBuckets: 8
-        scale: 1
-        growthFactor: 10
-    dimensions:
-      source: STRING
-      target: STRING
-      service: STRING
-      method: STRING
-      version: STRING
-      response_code: INT64
-
-# LOG
-- name: ApacheCommonLog-text
-  log_params:
-    description: Apache common log template
-    payloadFormat: TEXT
-    logTemplate: '{{or (.originIp) "-"}} - {{or (.sourceUser) "-"}} [{{or (.timestamp.Format "02/Jan/2006:15:04:05 -0700") "-"}}] "{{or (.method) "-"}} {{or (.url) "-"}} {{or (.protocol) "-"}}" {{or (.responseCode) "-"}} {{or (.responseSize) "-"}}'
-    dimensions:
-      originIp: IP_ADDRESS
-      sourceUser: STRING
-      timestamp: TIMESTAMP
-      method: STRING
-      url: STRING
-      protocol: STRING
-      responseCode: INT64
-      responseSize: INT64
 
 #
 ########################## contructors ########################
 #
 contructors:
 # QUOTA
 - instance_name: rl_by_service_by_consumer
```

Comment:
Yeah, constructors don't have names. Instances produced by the constructors are what are named.